tests/unit/providers/test_gemini_interactions.py

```
@@ -0,0 +1,469 @@
+
+
+@pytest.mark.parametrize(
+    "parameter",
+    ["temperature", "top_p", "max_tool_calls", "conversation", "prompt_cache_key"],
+)
+def test_convert_responses_params_rejects_unsupported(parameter: str) -> None:
+    value: Any
+    if parameter in {"conversation", "prompt_cache_key"}:
+        value = "c1"
+    else:
+        value = 1
```

Ruff (0.16.1)

[warning] Use ternary operator `value = "c1" if parameter in {"conversation", "prompt_cache_key"} else 1` instead of `if`-`else`-block

Replace `if`-`else`-block with `value = "c1" if parameter in {"conversation", "prompt_cache_key"} else 1`

(SIM108)